geopandas/array.py

```diff
@@ -769,9 +769,18 @@ def relate(self, other):
     #
 
     def unary_union(self):
+        warnings.warn(
+            "The 'unary_union' method is deprecated, "
```

Comment:
```suggestion
            "The 'unary_union' attribute is deprecated, "
```
Technically I think we should say property and note attribute, but attribute is consistent with the old deprecation for cascaded union.